packages/user-agent-parser/user-agent-parser.0.1.3/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Issuu Raccoons <racoon@issuu.com>"
+synopsis: "OCaml implementation of the user agent parse rules of uap-core"
+homepage: "https://github.com/issuu/uap-ocaml"
+doc: "https://issuu.github.io/uap-ocaml/"
+dev-repo: "git+https://github.com/issuu/uap-ocaml.git"
+bug-reports: "https://github.com/issuu/uap-ocaml/issues"
+build: [["dune" "build" "-p" name "-j" jobs]]
+run-test: [["dune" "runtest" "-p" name "-j" jobs]]
+depends: [
+  "alcotest" {with-test & >= "0.8.3"}
+  "base" {>= "v0.11"}
+  "dune"
```

Comment:
```suggestion
  "dune" {>="1.3"}
```

The dune-project indicates you need at least this version of dune, so adding bound.